docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md

```diff
@@ -1,9 +1,19 @@
 # CONFIG 语法
 
-**Nebula Graph** 使用 `gflags` 进行运行时配置。
+## 配置说明
+
+**Nebula Graph** 默认从 meta 获取配置。如需从本地获取配置，请在 flag 配置文件 `metad.conf`、 `storaged.conf`、`graphd.conf` （路径为 `/usr/local/nebula/etc/`）中分别添加 `local_config=true`，以从本地配置文件获取。
```

Comment:
In the new compiling build, the directory for config files is /home/user/nebula/build/install/etc